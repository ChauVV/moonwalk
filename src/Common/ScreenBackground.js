import React from "react";
import LinearGradient from "react-native-linear-gradient";
import styled from "styled-components";

const Wrapper = styled.View`
  background: #fff;
  flex: 1;
`;

export default ({ children, style }) => (
  // <LinearGradient
  //   colors={["#373468", "#222437"]}
  //   start={{ x: 0.0, y: 0.25 }}
  //   end={{ x: 0.5, y: 1.0 }}
  //   style={style}
  // >
  //   {children}
  // </LinearGradient>
  <Wrapper>{children}</Wrapper>
);
