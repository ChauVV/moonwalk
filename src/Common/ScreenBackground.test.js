import React from "react";
import ScreenBackground from "./ScreenBackground";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<ScreenBackground />).toJSON();
  expect(tree).toMatchSnapshot();
});
