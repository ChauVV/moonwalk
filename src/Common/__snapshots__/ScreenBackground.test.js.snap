// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screenBackground renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
      undefined,
    ]
  }
/>
`;
